client/src/components/History/Content/Dataset/DatasetMiscInfo.vue

```diff
@@ -0,0 +1,64 @@
+<script setup lang="ts">
+import { computed, ref, watch } from "vue";
+
+interface Props {
+    miscInfo: string;
+}
+
+const showErrorHelp = ref(false);
+const sharingError = ref(false);
+
+// old sharable error: Attempted to create shared output datasets in objectstore with sharing disabled
+// new sharable error: Job attempted to create sharable output datasets in a storage location with sharing disabled
```

Comment:
```suggestion
```

Reply:
I thought this was for regex debugging, but did you actually want to include it here for posterity?

Reply:
yeah - I thought it would be good for future austerity so we can keep ensuring this older message is caught and handled appropriately.